feat(refactor): Basehandle

henrikstranneheim opened the pull request:
## Description

Refactor Basehandler initialization.

### Added

-

### Changed

- Remove store/api.py
- Extract database initialization to corehandler 
- Rename Basehandler_2 to Basehandler

### Fixed

-

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] activate stage: `us`
- [ ] request trailblazer-stage on hasta: `paxa`
- [ ] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b refactor_basehandler -a
    ```
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

islean reviewed (approved):
LGMT! 🔥

ChrOertlin reviewed (approved):
👍 Looks good

diitaz93 reviewed (approved):
Great! 👍

seallard commented on trailblazer/store/core.py:
```diff
@@ -1,7 +1,10 @@
+import alchy
```
How come we are adding `alchy`?

seallard replied:
Oh, it used to be in `api.py`

seallard reviewed (approved):
Nice, looks good to me

henrikstranneheim commented:
### CLI and API ✅ 

```
θ60° [henrik.stranneheim@hasta:~] [S_base] 8m56s $ trailblazer scan
Error updating analysis for: case - grandaphid : PermissionError
Error updating analysis for: case - goodbengal : MissingSqueueOutput
Error updating analysis for: case - helpedlab : MissingSqueueOutput
Error updating analysis for: case - newgopher : PermissionError
Error updating analysis for: case - onestag : PermissionError
Error updating analysis for: case - funnyaphid : FileNotFoundError
Error updating analysis for: case - proudfish : PermissionError
Error updating analysis for: case - setcolt : PermissionError
Error updating analysis for: case - handyshrimp : FileNotFoundError
Error updating analysis for: case - divinetomcat : PermissionError
Error updating analysis for: case - HW2V2DRX2 : FileNotFoundError
Error updating analysis for: case - savingstag : FileNotFoundError
Error updating analysis for: case - topewe : PermissionError
Error updating analysis for: case - fullgoat : PermissionError
Error updating analysis for: case - bolddrum : PermissionError
Error updating analysis for: case - abovehusky : PermissionError
Error updating analysis for: case - chiefgannet : PermissionError
Error updating analysis for: case - grandoyster : PermissionError
Status in Tower: coolkodiak - 282044 - WPjrjs23yxQwq
Error logging case - coolkodiak :  JSONDecodeError
Updated status amazedbaboon - 282547: running
Updated status savedkid - 282549: running
Updated status coolbug - 282550: running
Error updating analysis for: case - merrydoe : PermissionError
Updated status newmaggot - 282552: running
Updated status steadyfoal - 282553: running
Error updating analysis for: case - secureosprey : PermissionError
Updated status setturtle - 282555: running
Error updating analysis for: case - crackfowl : PermissionError
Updated status masterrat - 282557: running
Error updating analysis for: case - politerabbit : PermissionError
Updated status loyalpony - 282559: running
Error updating analysis for: case - crispbaboon : PermissionError
Updated status rightdory - 282561: running
Error updating analysis for: case - honestbug : PermissionError
Updated status setibex - 282563: running
Error updating analysis for: case - validgoblin : PermissionError
Updated status superbmacaw - 282565: running
Updated status justkoi - 282566: running
Error updating analysis for: case - bigkoala : PermissionError
Updated status proadder - 282568: running
Error updating analysis for: case - crispturkey : PermissionError
Updated status mightychow - 282570: running
Error updating analysis for: case - primedeer : PermissionError
Updated status choicecattle - 282572: running
Error updating analysis for: case - modernamoeba : PermissionError
Updated status mastermutt - 282575: running
Error updating analysis for: case - finekitten : PermissionError
Updated status likedcod - 282577: running
Error updating analysis for: case - handyperch : PermissionError
Updated status choiceraven - 282579: running
Updated status largesalmon - 282580: running
Updated status grandpup - 282581: running
Updated status uniquewombat - 282582: running
Updated status selectbass - 282583: running
Updated status brightwhale - 282584: running
Updated status fairmolly - 282585: running
Updated status soughtbengal - 282586: running
Error updating analysis for: case - starbadger : PermissionError
Error updating analysis for: case - niceelk : PermissionError
Updated status usableghoul - 282590: failed
Error updating analysis for: case - eagerthrush : PermissionError
Error updating analysis for: case - clearmarten : PermissionError
Updated status abovebee - 282593: running
Error updating analysis for: case - restedhen : PermissionError
Error updating analysis for: case - directteal : PermissionError
Updated status chiefweasel - 282597: running
Error updating analysis for: case - oneshark : PermissionError
Error updating analysis for: case - fondhawk : PermissionError
Updated status finerheron - 282600: running
Updated status cutebird - 282601: running
Updated status alertfilly - 282602: running
Error updating analysis for: case - frankamoeba : PermissionError
Error updating analysis for: case - amazedalien : PermissionError
Error updating analysis for: case - opencoyote : PermissionError
Error updating analysis for: case - directskink : PermissionError
Error updating analysis for: case - modeldrum : PermissionError
Error updating analysis for: case - savedtoad : PermissionError
Error updating analysis for: case - safevervet : PermissionError
Error updating analysis for: case - heroicguppy : PermissionError
Error updating analysis for: case - wiredghoul : PermissionError
Error updating analysis for: case - usablestork : PermissionError
Error updating analysis for: case - hiprobin : PermissionError
Error updating analysis for: case - readyboar : PermissionError
All analyses updated!
```

<img width="1727" alt="Screenshot 2023-11-09 at 10 36 05" src="https://github.com/Clinical-Genomics/trailblazer/assets/584738/122d19f0-b0ba-45e6-b7ad-79e7363d956f">

henrikstranneheim commented:
Deployed in production:

```
Log deploy... done.
trailblazer, version 19.0.2
```
